apps/andi/test/unit/andi_web/live/access_group_live_view/user_table_test.exs

```diff
@@ -0,0 +1,102 @@
+defmodule AndiWeb.AccessGroupLiveView.UserTableTest do
+  use AndiWeb.Test.AuthConnCase.UnitCase
+  use Placebo
+  alias Andi.Schemas.User
+
+  import Phoenix.LiveViewTest
+  import FlokiHelpers, only: [get_text: 2]
+
+  alias SmartCity.TestDataGenerator, as: TDG
+  alias Andi.InputSchemas.AccessGroups
+  alias Andi.InputSchemas.AccessGroup
+
+  @endpoint AndiWeb.Endpoint
+  @url_path "/access-groups"
+  @user UserHelpers.create_user()
+
+  defp allowAuthUser do
+    allow(Andi.Repo.get_by(Andi.Schemas.User, any()), return: @user)
+    allow(User.get_all(), return: [@user])
+    allow(User.get_by_subject_id(any()), return: @user)
+  end
+
+  setup do
+    allowAuthUser()
+    []
+  end
+
+  setup %{auth_conn_case: auth_conn_case} do
+    auth_conn_case.disable_revocation_list.()
+    :ok
+  end
+
+  describe "Basic associated users table load" do
+    test "shows \"No Associated Users\" when there are no rows to show", %{conn: conn} do
+      access_group = setup_access_group()
+      allow(Andi.Repo.preload(any(), any()), return: %{datasets: [], users: []})
+
+      assert {:ok, view, html} = live(conn, "#{@url_path}/#{access_group.id}")
```

Comment:
non blocking:

is there a technical reason why `live` made more sense here than `live_isolated`?  Hoping to gain more perspective on which one makes sense in what scenarios. (not worth re-writing or anything like that, just curious!)

Reply:
We did this because the event handlers are on the parent component, so live_isolated seemed to not be the best fit!

Reply:
Oh yep. I mentioned that that would be a tell that `live` would be necessary when we found `live_isolated`

thanks for the explanation 👍